### ISSUE TITLE
Passing path for dataloader in windows

### ISSUE THREAD

**Issue by zama-sarib** [Deci-AI/super-gradients#1293]
### 💡 Your Question

How do we pass the dataset path and train, val and test path to the dataloader on windows ? Can somebody guide or better if can share the code .
My Folder Structure:

ProjectName 
 |
 |
DataSet
  |
  |__Train
  |      |__Images
  |      |__Labels
  |
  |__Val
  |     |__Images
  |     |__Labels
  |
  |___Test
        |__Images
        |__Labels
